arctic/store/_ndarray_store.py

```diff
@@ -120,6 +129,21 @@ def set_corruption_check_on_append(enable):
     _CHECK_CORRUPTION_ON_APPEND = bool(enable)
 
 
+def update_fw_pointers(collection, result, symbol, version, previous_version):
+    if result is None:
+        return
+    if previous_version and FW_POINTERS_KEY not in previous_version:
+        chunk_ids = [_id for _id in collection.find({'symbol': symbol, 'parent': version_base_or_id(version)},
+                                                    {'_id': 1})]
+    else:
+        chunk_ids = previous_version[FW_POINTERS_KEY] if previous_version else []
+    upserted_ids = [result.upserted_id] \
```

Comment:
why use the ternary operator and escape newlines instead of a bog standard if?

Reply:
why even assign to upserted_ids?

Reply:
should be more readable now, tks